addons/website_blog/views/website_blog_templates.xml

```diff
@@ -406,146 +415,158 @@
+                    <a t-att-href="website.social_github" t-if="website.social_github" aria-label="Github" title="Github"><i class="fa fa-github"/></a>
+                    <a t-att-href="website.social_instagram" t-if="website.social_instagram" aria-label="Instagram" title="Instagram"><i class="fa fa-instagram"/></a>
+                    <a t-att-href="'/blog/%s/feed' % (blog.id)" aria-label="RSS" title="RSS"><i class="fa fa-rss-square"/></a>
+                </h2>
+                <hr/>
+                <t t-if="blog">
+                    <t t-call="website_mail.follow">
+                        <t t-set="email" t-value="user_id.email"/>
+                        <t t-set="object" t-value="blog"/>
+                        <t t-set="div_class" t-value="'mx-auto'"/>
+                    </t>
                 </t>
-            </t>
-            <p class="text-muted mb0 mt16">
-                Participate on our social stream.
-            </p>
-            <h2 class="mt4">
-                <a t-att-href="website.social_facebook" t-if="website.social_facebook" aria-label="Facebook" title="Facebook"><i class="fa fa-facebook-square"/></a>
-                <a t-att-href="website.social_twitter" t-if="website.social_twitter" aria-label="Twitter" title="Twitter"><i class="fa fa-twitter"/></a>
-                <a t-att-href="website.social_linkedin" t-if="website.social_linkedin" aria-label="LinkedIn" title="LinkedIn"><i class="fa fa-linkedin"/></a>
-                <a t-att-href="website.social_youtube" t-if="website.social_youtube" aria-label="Youtube" title="Youtube"><i class="fa fa-youtube-play"/></a>
-                <a t-att-href="website.social_googleplus" t-if="website.social_googleplus" aria-label="Google Plus" title="Google Plus"><i class="fa fa-google-plus-square"/></a>
-                <a t-att-href="website.social_github" t-if="website.social_github" aria-label="Github" title="Github"><i class="fa fa-github"/></a>
-                <a t-att-href="website.social_instagram" t-if="website.social_instagram" aria-label="Instagram" title="Instagram"><i class="fa fa-instagram"/></a>
-                <a t-att-href="'/blog/%s/feed' % (blog.id)" aria-label="RSS" title="RSS"><i class="fa fa-rss-square"/></a>
-            </h2>
-        </section>
+            </div>
+        </div>
     </xpath>
 </template>
 
 <!-- Option:Right Column: status -->
 <template id="opt_blog_rc_status" name="Status" priority="5"
         inherit_id="website_blog.index_right" active="True" customize_show="True">
     <xpath expr="//div[@id='main_column']" position="attributes">
-        <attribute name="class">col-md-8</attribute>
+        <attribute name="class">col</attribute>
     </xpath>
     <xpath expr="//div[@id='blog_right_column']" position="inside">
-        <section class="mt32 mb32" groups="website.group_website_designer">
```

Comment:
@kea14 I am working on this task. Comment about this (do not fix as I take care of this branch now): be careful, you removed a `groups` here, which means that all visitors would have had access to this panel :|